docs/_documentations/remote-deploying-codewind.md

```diff
@@ -59,22 +65,55 @@ Add the option `-o` flag if you are installing into an OpenShift 3.11 cluster, f
-
-The command returns the following example output:
+The script installs the operator into your cluster, deploys Keycloak, and returns the Keycloak **Access URL**. The command returns the following example output:
 
 ```
 NAME       NAMESPACE   AGE    ACCESS
 devex001   codewind    122m   https://codewind-keycloak-devex001.<ingress-domain>
 ```
 
+### 2b. Change the admin password for the admin user
+
+You must prepare Keycloak for Codewind by changing the admin password given to the `admin` user in the previous step. 
+
+1. Access the Administration Console by clicking the **Access URL** link provided by Step 2a. If you cannot locate the **Access URL** link, enter the `$ kubectl get keycloaks -n codewind` command. 
+
+2. Log in to Keycloak using the Keycloak admin credentials.
+        username: `admin`
+        password: `admin`
+
+3. **Important:**: After you log in, change the admin password by clicking the **Admin** link on the page. Then choose **Manage Account / Password** and set a new replacement administrator password.
+
+4. Switch back to the admin console using the link or log out and log back in to Keycloak as the admin user with your new admin password.
+
+You have updated the Keycloak password. The next step updates the Keycloak password in the operator secret to match it. 
+
+### 2c. Update the Keycloak password in the operator secret with the new password
+
+When the Codewind Operator needs to update Keycloak, it uses login credentials saved in a Kubernetes secret. By default during initial deployment, that secret has a user name and password of **admin**. If you changed your admin password in a previous step, you need to update the Keycloak secret to match.
+
+The secret is installed in the same namespace as the codewind operator and is named `secret-keycloak-user-{keycloakname}`.
+
+If you have an administration UI for your cluster, you can use it to locate the secret and edit the `keycloak-admin-password` field, or you can use the command line tools:
+
+`$ kubectl edit secret secret-keycloak-user-{keycloakname} -n codewind`
+
+or
+
+`$ oc edit secret secret-keycloak-user-{keycloakname} -n codewind`
+
+Note: Using the command line tools requires an extra step to base64 encode your password string before saving it into the secret. You can base64 encode your new password using this command:
+
+```bash
+$ echo -n 'myNewPassword' | base64
+bXlOZXdQYXNzd29yZA==
+```
+
+Then, save `bXlOZXdQYXNzd29yZA==` as the value for `keycloak-admin-password` rather than the clear text `myNewPassword`.
+
 ## 3. Add a new user to Keycloak
 
-1. Copy the **Access URL** and paste it into a browser.
+1. Access the Administration Console by clicking the **Access URL** link provided by Step 2a. If you cannot locate the **Access URL** link, enter the `$ kubectl get keycloaks -n codewind` command. 
 
 2. Click **Administration Console** and log in to Keycloak with `admin` as the default login user name and `admin` as the default password.
```

Comment:
Should we mention to use the "NEW" password you set in step 2b rather than the default password of "admin"

Reply:
Good catch, have updated the text.